osrf_testing_tools_cpp/include/osrf_testing_tools_cpp/memory_tools/register_hooks.hpp

```diff
@@ -65,6 +70,11 @@ OSRF_TESTING_TOOLS_CPP_MEMORY_TOOLS_PUBLIC
 void
 on_realloc(AnyMemoryToolsCallback callback);
 
+/// Get the current on_realloc callback if set, otherwise null.
+OSRF_TESTING_TOOLS_CPP_MEMORY_TOOLS_PUBLIC
+AnyMemoryToolsCallback
+get_on_realloc();
```

Comment:
Given that we don't actually use `get_on_realloc`/`get_on_calloc`/`get_on_free`, should we bother exposing them?  I generally like to only expose the things that I'm using, to keep the API smaller.